web/src/PickNeighbourhoodMode.svelte

```diff
@@ -79,6 +80,26 @@
     );
   }
 
+  function countEdits(gj: FeatureCollection): {
```

Comment:
Being sloppier, we'd just have a mapping from string to a counter (auto-set to 0) per `kind` property, but I couldn't get this all nicely working with TS, so verbose for now